apps/wallet-mobile/src/WalletInit/ImportReadOnlyWallet/ImportReadOnlyWalletScreen.tsx

```diff
@@ -41,7 +41,7 @@ export const ImportReadOnlyWalletScreen = () => {
       <StatusBar translucent backgroundColor="transparent" />
 
       <View style={styles.cameraContainer}>
-        <QRCodeScanner onRead={onRead} />
+        <QRCodeScanner onRead={onRead} maskEnabled={false} />
```

Comment:
Boolean props make it optional.
```suggestion
        <QRCodeScanner onRead={onRead} />
```